lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -5314,4 +5320,16 @@ private void makeSamplePeerEval(UIContainer parent)
 		UIOutput.make(peerReviewRows, "peer-eval-sample-text", messageLocator.getMessage("simplepage.peer-eval.sample.4"));
 	}
 
+	private String personalizeText(String itemText) {
+		if (StringUtils.isNotBlank(itemText)) {
+			User currentUser = UserDirectoryService.getCurrentUser();
+			if (currentUser != null) {
```

Comment:
Very true.

I like the idea of specifying a default, but I agree that it's probably overkill for this. I've pushed the requested changes.